### ISSUE TITLE
Text Editor reset issue.

### ISSUE THREAD

**Issue by sachinchoudhary61** [AmruthPillai/Reactive-Resume#1991]
### Is there an existing issue for this?

- [X] Yes, I have searched the existing issues and none of them match my problem.

### Product Variant

Cloud (https://rxresu.me)

### Current Behavior

When I click on custom field section / Any - field section. So , the summary section for that field is not getting reset , 

### Expected Behavior

it should reset . you should reset the value when I  click on another filed , because every field / section has its own sense 
So. , The previous saved text in the previous working field on navigating it should reset. 

### Steps To Reproduce

Click on the Builder section 
Click on any resume field / custom field 
add text in summary 
save it 
navigate to another filed 
on navigation u can see the text within  it.

### What browsers are you seeing the problem on?

Chrome

### What template are you using?

Rhyhorn

**Comment by Mahesh-1919** [comment]
hello @sachinchoudhary61 @AmruthPillai   , is this issue still open??
can i try to solve this issue ??

**Comment by sachinchoudhary61** [comment]
yes @Mahesh-1919 , u can try to resolve this issue u just need to reset the variable that u are using here , on click event 
If u have any questions just let me know 
& This is still open first try to reproduce at ur end , then we can discuss further if required

**Comment by amruthpillai** [comment]
Closing this issue as I'm doing some clean up to move to a newer version of the project. I apologize in advance for the inconvenience if you find the need to reopen this issue.

Please read #2499 for more information about the migration.
